README.md

```diff
@@ -44,15 +44,15 @@ The TestCafe version to install.
     args: "chrome tests"
 ```
 
-The **latest version** is installed by default.
+**Default value:** `latest` (the latest version)
 
 ## Examples
 
 This section contains sample workflows that show how to use `testcafe-action`.
 
 ### Run TestCafe Tests
 
-The following workflow demonstrates the basic usage of `testcafe-action`.
+The following workflow demonstrates the basic `testcafe-action` use case.
```

Comment:
"basic `testcafe-action` use case" Why 'basic use case'? Does it show hoe to use it? How it works?